packages/devtools_app/lib/src/sliver_iterable_child_delegate.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/widgets.dart';
```

Comment:
Fyi @kenzieschmoll @terrylucas. This could be handy other cases in DevTools as well.